runelite-client/src/main/java/net/runelite/client/plugins/npcunaggroarea/NpcAggroAreaPlugin.java

```diff
@@ -470,6 +503,8 @@ public void onGameStateChanged(GameStateChanged event)
 					onLogin();
 				}
 
+				npcLootCount = 0;
+				previousNPCName = null;
```

Comment:
This would reset the counter if a player hopped worlds or disconnected, which seems like the wrong time to do that.

Reply:
You're correct - doesn't make sense to clear on a world hop.  Probably me still learning how the framework works; I misunderstood when that would be called.

Reply:
I've pushed the change up.

Reply:
It doesn't look like you have

Reply:
Done